.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
     - hypothesis
     - future
     - cloudpickle
-    - gym
+    - gym==0.25.1
```

Comment:
we should be able to remove this now

Reply:
The main branch currently has it fixed too, should I remove it here?